keps/sig-auth/4633-anonymous-auth-configurable-endpoints/README.md

```diff
@@ -616,6 +623,8 @@ enhancement:
   CRI or CNI may require updating that component before the kubelet.
 -->
 
+N/A
```

Comment:
PRR reviewers will ask you for this, specially if you specify that two clusters with the feature gate on and off will behave differently

Reply:
Added. Thanks!